cli_tools_tests/e2e/gce_image_import_export/test_suites/import/cli_tests.go

```diff
@@ -364,14 +379,96 @@ func runImageImportShadowDiskCleanedUpWhenMainInflaterFails(ctx context.Context,
 	}
 }
 
+func runImageImportOSDefaultServiceAccountWithMissingPermissionsTest(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
```

Comment:
It's not clear from either test case names or this func names if this should suceed or fail. E.g. missing permissions should probably fail but I don't see any assertion for test failure.

Reply:
Related: I'm thinking for each tool, you'd want coverage for all of these combinations (at least the first two, but for full coverage the third as well):

- Service account specified:  default, custom
- Does the specified account exist?
- Does the specified account have sufficient permissions?

Some will generate error messages, and that's good, since it will allow you to assert the error message that is returned.

Reply:
I'm glad we have the core scenarios. For the error message scenarios, I'm fine classifying those as "nice to have"